mobile/mobile-core/android-process-service.ts

```diff
@@ -285,7 +286,12 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 				// We need to use shelljs here instead of $adb because listener functions of exit, SIGINT and SIGTERM must only perform synchronous operations.
 				// The Node.js process will exit immediately after calling the 'exit' event listeners causing any additional work still queued in the event loop to be abandoned.
 				// See the official documentation for more information and examples - https://nodejs.org/dist/latest-v6.x/docs/api/process.html#process_event_exit.
-				shelljs.exec(`adb forward --remove tcp:${port}`);
+
+				try {
+					shelljs.exec(`adb forward --remove tcp:${port}`);
```

Comment:
This code is used only in Proton and here `shelljs` will not throw error and we do not need this `try/catch`